pkg/registry.go

```diff
@@ -121,6 +125,20 @@ type RegistryConfig struct {
 	ClientTimeout                   int
 }
 
+func DefaultRegistryConfig() RegistryConfig {
+	return RegistryConfig{
+		Mode:                            core.ServerEngineMode,
+		SyncMode:                        "fs",
+		SyncAddress:                     "https://codeload.github.com/nuts-foundation/nuts-registry-development/tar.gz/master",
+		SyncInterval:                    30,
+		Datadir:                         "./data",
+		Address:                         "localhost:1323 ",
+		VendorCACertificateValidity:     1095,
```

Comment:
use values from defs?

Reply:
They're the same (cmd defaults use DefaultRegistryConfig() func)